packages/plugin-react/src/index.ts

```diff
@@ -278,7 +278,8 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
         if (shouldSkip) {
           // Avoid parsing if no plugins exist.
           return {
-            code
+            code,
+            map: null
```

Comment:
Should we call `transformStableResult` here? https://github.com/vitejs/vite/blob/main/packages/vite/src/node/utils.ts#L1048

So sourcemaps also get the column right during build time? Or is `code` always guaranteed to be unmodified here? In which case we could return `undefined` to make that more explicit, no?

Reply:
Actually this line was changing the code. 😅 Thanks.
https://github.com/vitejs/vite/blob/ac694a4979574f90d66bd9eb0c7f9d2fbd3abc57/packages/plugin-react/src/index.ts#L264